lib/envs.js

```diff
@@ -138,6 +139,20 @@ function loadEnvs(path) {
   return env;
 }
 
+function loadEnvs(path) {
+  if (path.indexOf(',') === -1) {
```

Comment:
You're correct. Was keeping the existing behavior when there's no `','` for a slight performance gain. It's negligible though, so I can remove it if that's preferred.